src/components/Equations.ts

```diff
@@ -104,22 +105,25 @@ export class Equations<TGameStartr extends FullScreenPokemon> extends Component<
      *             from the newPokemonEVs equation).
      * @returns A newly created Pokemon.
      */
-    public newPokemon(title: string[], level?: number, moves?: IMove[]): IPokemon {
+    public newPokemon(chosenInfo: INewPokemon): IPokemon {
         const ev: IValuePoints = this.newPokemonEVs();
         const iv: IValuePoints = this.newPokemonIVs();
-        level = level || 1;
+        const level = chosenInfo.level || 1;
+        const item = chosenInfo.item;
+        const title = chosenInfo.title;
 
         return {
             experience: this.experienceStarting(title, level || 1),
             ev,
+            item: item || undefined,
```

Comment:
You set `item` above then use it only once here. Inline the usage for fewer lines of code.
Same with `level` and `title`.